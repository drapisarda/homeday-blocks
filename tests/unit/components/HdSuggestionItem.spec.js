import { mount } from '@vue/test-utils';
import HdSuggestionItem from '@/components/HdSuggestionItem.vue';

describe('HdSuggestionItem', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(HdSuggestionItem, {
      propsData: {
        line1: 'Lorem ipsum dolor sit amet',
        line2: 'onsectetur adipiscing elit',
        icon: 'https://dummyimage.com/32x32',
        highlightString: 'consectetur',
        isSelected: true,
      },
    });
  });

  test('renders component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
