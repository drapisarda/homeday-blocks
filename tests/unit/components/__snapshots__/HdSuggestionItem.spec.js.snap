// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HdSuggestionItem renders component 1`] = `
<div class="suggestionItem suggestionItem--isSelected"><img src="https://dummyimage.com/32x32" class="suggestionItem__icon">
  <div class="suggestionItem__content">
    <p class="suggestionItem__line suggestionItem__line--1">Lorem ipsum dolor sit amet</p>
    <p class="suggestionItem__line suggestionItem__line--2">onsectetur adipiscing elit</p>
  </div>
</div>
`;
